downloader/history_downloader/__init__.py

```diff
@@ -0,0 +1,12 @@
+import ccxt
+
+class HistoryDownloader:
+    exchange = ccxt.binance() #creating instance of binance exchange

```

Comment:
This kind of variable will be treated as a class scope variable and will be the same in every `HistoryDownloader` class instance (object). Are you sure this is the right way to define universal `HistoryDownloader` class?
`HistoryDownloader` should rather be exchange independent as a user wants to download data from few different exchanges at the same time and he/she want to initialize this class more than once.

Reply:
Alright, I thought that we will use only the Binance exchange. I should change it to allow the user to pick any exchange provided in ccxt library?